package-lock.json

```diff
@@ -1878,6 +1878,7 @@
 			"version": "10.0.19",
 			"resolved": "https://registry.npmjs.org/@emotion/cache/-/cache-10.0.19.tgz",
 			"integrity": "sha512-BoiLlk4vEsGBg2dAqGSJu0vJl/PgVtCYLBFJaEO8RmQzPugXewQCXZJNXTDFaRlfCs0W+quesayav4fvaif5WQ==",
+			"dev": true,
```

Comment:
If there are no changes to a `package.json`, it's not clear to me why this file would have been changed?

Reply:
@aduth I'm not sure 🙈 . I think it had to do with my `npm` version. I'll revert this to the one from `master`.